tests/testthat/test-compute_scale.R

```diff
@@ -0,0 +1,155 @@
+# Input test dataset
+adqs <- tibble::tribble(
+  ~STUDYID, ~USUBJID, ~PARAMCD, ~AVISIT, ~AVISITN, ~AVAL,
+  "ADMIRAL01", "01-701-1015", "ITEM1", "WEEK 10", 100, 5,
+  "ADMIRAL01", "01-701-1015", "ITEM2", "WEEK 10", 100, 1,
+  "ADMIRAL01", "01-701-1015", "ITEM3", "WEEK 10", 100, 2,
+  "ADMIRAL01", "01-701-1015", "ITEM4", "WEEK 10", 100, NA,
+  "ADMIRAL01", "01-701-1015", "ITEM1", "WEEK 20", 200, 1,
+  "ADMIRAL01", "01-701-1015", "ITEM2", "WEEK 20", 200, 3,
+  "ADMIRAL01", "01-701-1015", "ITEM3", "WEEK 20", 200, 5,
+  "ADMIRAL01", "01-701-1015", "ITEM4", "WEEK 20", 200, 5,
+  "ADMIRAL01", "01-701-1015", "ITEM5", "WEEK 20", 200, 1,
+  "ADMIRAL01", "01-701-1015", "ITEM6", "WEEK 20", 200, 3,
+  "ADMIRAL01", "01-701-1015", "ITEM7", "WEEK 20", 200, 3,
+  "ADMIRAL01", "01-701-1281", "ITEM1", "WEEK 10", 100, 4,
+)
+
```

Comment:
@bundfussr I think we stopped putting the inputs outside of `test_that` calls as this can have unintended side-effects, i.e. I think this gets stored in the global environment.  Shall we bring down into each test?

It looks like each input for the tests is actually just a vector of the adqs dataset??  Maybe the dataset is not needed?